arbeitszeit_flask/templates/member/notification-about-work-invitation.html

```diff
@@ -0,0 +1,6 @@
+<p>{{
+    gettext(
+        'You have been invited by a company to join them on Arbeitszeitapp. Log in as a member and find the invitation on your dashboard.'
```

Comment:
I think that this email would be 10x more useful if it would include a link to the invitiation. The UrlIndex already has a method the generate the appropriate link addresses for that purpose (`get_answer_company_work_invite_url`).